File: Blackoffice/Models/MAS_Menu_Permis_Models.vb
Imports MD
Public Class MAS_Menu_Permis_Models
    Public Class value_menu
        Public Respon As New ResponseModels
        Public Data As List(Of Data_Menu)
    End Class
    Public Class value_List
        Public Respon As New ResponseModels
        Public Data As List(Of Data_List)
    End Class
    Public Class value_List_CBO
        Public Respon As New ResponseModels
        Public Data As List(Of Data_List_CBO)
    End Class
    Public Class value_SAVE
        Public Respon As New ResponseModels
        Public Data As New Data_SAVE

    End Class
    Public Class Data_SAVE
        Public Data_H As New Data_PER_MIS
        Public Data_ITEMS As New List(Of Data_PER_MIS_D)

    End Class
    Public Class Data_Menu

        Public Property ID As Integer
        Public Property MNU_CODE As String
        Public Property MNU_HSEQ As Integer
        Public Property MNU_HNAME_1 As String
        Public Property MNU_HNAME_2 As String
        Public Property MNU_SSEQ As Integer
        Public Property MNU_SNAME_1 As String
        Public Property MNU_SNAME_2 As String
        Public Property MNU_TAG As String
        Public Property MNU_OBJ1 As String
        Public Property MNU_OBJ2 As String
        Public Property MNU_ICO As String
        Public Property MNU_DEFAULT_DOCTYPE As String
        Public Property U_DATE As DateTime
        Public Property U_ID As String
        Public Property ISTAT As String

    End Class
    Public Class Data_List_CBO

        Public Property TR_ID As String
        Public Property PERMIS_CODE As String
        Public Property PERMIS_NAME As String


    End Class
    Public Class Data_List

        Public Property TR_ID As String
        Public Property PERMIS_CODE As String
        Public Property PERMIS_NAME As String
        Public Property UID As String
        Public Property ACTIVEFLAG As String
        Public Property MDATE As DateTime
        Public Property MUSE As String


    End Class
    Public Class Data_PER_MIS
        Public Property ID As Integer
        Public Property TR_ID As String
        Public Property PERMIS_CODE As String
        Public Property PERMIS_NAME As String
        Public Property UID As String
        Public Property DESCRIPTION As String
        Public Property REMARK As String
        Public Property ACTIVEFLAG As String
        Public Property [CDATE] As DateTime
        Public Property CUSE As String
        Public Property MDATE As DateTime
        Public Property MUSE As String
        Public Property ISTAT As String


    End Class
    Public Class Data_PER_MIS_D
        Public Property TR_ID As String
        Public Property PERMIS_CODE As String
        Public Property SEQ As Integer
        Public Property MNU_CODE As String
        Public Property MNU_HNAME_1 As String
        Public Property MNU_SNAME_1 As String
        Public Property M_AT As Boolean
        Public Property BTN_ADD As Boolean
        Public Property BTN_EDIT As Boolean
        Public Property BTN_VIEW As Boolean
        Public Property BTN_DEL As Boolean
        Public Property REMARK As String

    End Class
End Class

File: TouchPOS_API/Models/MAS_Menu_Permis_Models.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using BaseRepository.Helpers;
using TouchPOS_API.Helpers;
using System.Configuration;


namespace TouchPOS_API.Models
{
    public class MAS_Menu_Permis_Models
    {
        public class PER_MIS_Save
        {
            public Data_PER_MIS Data_H = new Data_PER_MIS();
            public List<Data_PER_MIS_D> Data_ITEMS = new List<Data_PER_MIS_D>();


        }
        public class Data_Menu
        {

            [Column]
            public int ID { get; set; }
            [Column]
            public string MNU_CODE { get; set; }
            [Column]
            public int MNU_HSEQ { get; set; }
            [Column]
            public string MNU_HNAME_1 { get; set; }
            [Column]
            public string MNU_HNAME_2 { get; set; }
            [Column]
            public int MNU_SSEQ { get; set; }
            [Column]
            public string MNU_SNAME_1 { get; set; }
            [Column]
            public string MNU_SNAME_2 { get; set; }
            [Column]
            public string MNU_TAG { get; set; }
            [Column]
            public string MNU_OBJ1 { get; set; }
            [Column]
            public string MNU_OBJ2 { get; set; }
            [Column]
            public string MNU_ICO { get; set; }
            [Column]
            public string MNU_DEFAULT_DOCTYPE { get; set; }
            [Column]
            public DateTime U_DATE { get; set; }
            [Column]
            public string U_ID { get; set; }
            [Column]
            public string ISTAT { get; set; }

          
            
        }
        public class Data_List_CBO
        {
            public string TR_ID { get; set; }
            [Column]
            public string PERMIS_CODE { get; set; }
            [Column]
            public string PERMIS_NAME { get; set; }

        }
        public class Data_List
        {
            public string TR_ID { get; set; }
            [Column]
            public string PERMIS_CODE { get; set; }
            [Column]
            public string PERMIS_NAME { get; set; }
            [Column]
            public string UID { get; set; }
            [Column]
            public string ACTIVEFLAG { get; set; }       
            [Column]
            public DateTime MDATE { get; set; }
            [Column]
            public string MUSE { get; set; }


        }
        [Table("PER_MIS")]
        public class Data_PER_MIS
        {


            [Column]
            public int ID { get; set; }
            [Column]
            public string TR_ID { get; set; }
            [Column]
            public string PERMIS_CODE { get; set; }
            [Column]
            public string PERMIS_NAME { get; set; }
            [Column]
            public string UID { get; set; }
            [Column]
            public string DESCRIPTION { get; set; }
            [Column]
            public string REMARK { get; set; }
            [Column]
            public string ACTIVEFLAG { get; set; }
            [Column]
            [Date(Now = true)]
            public DateTime CDATE { get; set; }
            [Column]
            public string CUSE { get; set; }
           [ Column]
            [Date(Now = true)]
            public DateTime MDATE { get; set; }
            [Column]
            public string MUSE { get; set; }
            [Column]
            public string ISTAT { get; set; }

        }
        [Table("PER_MIS_D")]
        public class Data_PER_MIS_D
        {
            [Column]
            public string TR_ID { get; set; }
            [Column]
            public string PERMIS_CODE { get; set; }
            [Column]
            public int SEQ { get; set; }
            [Column]
            public string MNU_CODE { get; set; }
             [Column]
            public string MNU_HNAME_1 { get; set; }
            [Column]
            public string MNU_SNAME_1 { get; set; }

            [Column]
            public bool M_AT { get; set; }
            [Column]
            public bool BTN_ADD { get; set; }
            [Column]
            public bool BTN_EDIT { get; set; }
            [Column]
            public bool BTN_VIEW { get; set; }
            [Column]
            public bool BTN_DEL { get; set; }
            [Column]
            public string REMARK { get; set; }

        }
    }
}